### PR TITLE
finish game implementation

### PR DESCRIPTION
**Pull request by YESENIA2012**
Game implementation:

https://user-images.githubusercontent.com/68444001/146655982-e7c86003-8815-4d32-bde4-5907089575ee.mov

* The inline-block display was removed from the row class and placed in the column class
* Add h3 tag in html file to show winner player
* A js file called utils was created
* the drawGame function was added inside the utils file to draw the game board
* three functions ( findWinnerInDiagonals, findWinnerInRows, findWinnerInColumns) were created to find the winner in the utils file.

### CODE REVIEW

**Comment by YESENIA2012** [comment]
* change 1: The inline-block display was removed from the row class and placed in the column class
* change 2: add h3 tag in html file to expose winning player
* change 3: a js file called utils was created, where the function that draws the game table is found and is inserted into the html
* change 4: in the utils file a function is implemented that goes through the diagonals and looks for if there is a winner in them
* change 5: in the utils file a function is implemented that goes through the rows and searches if there is a winner in them
* change 6: in the utils file a function is implemented that goes through the columns and looks for if there is a winner in them
* change 6: the name of the array is changed to scores

File: creatingGame.js [hunk 1]
```diff
@@ -16,46 +16,59 @@ function matriz() {
...
-      columnElement.setAttributeNode(atributo2);
-      let imagen = document.createElement("img");
-      columnElement.appendChild(imagen);
-      rowElement.appendChild(columnElement);
-    }
-  }
+  let numRows = document.getElementById("filas").value;
+  let numColumns = document.getElementById("columnas").value;
+
+  let game = drawGame(numRows, numColumns);
```
Comment (erick2014) [inline creatingGame.js]: rename the `game` variable to `dashboardGame` . Also rename the variable on lines 23, 24

File: creatingGame.js [hunk 2]
```diff
@@ -16,46 +16,59 @@ function matriz() {
...
   let padre = document.querySelector(".padre");
   padre.addEventListener("click", insertarX);
 
   var turno = jugador1;
+  let playCounter = 0;
 
   function insertarX(evento) {
+    playCounter = playCounter + 1;
     let imagen = evento.target;
+    let atributosImagen = imagen.getAttribute("class");
```
Comment (erick2014) [inline creatingGame.js]: this variable, should be const, and should be translate to english: `imageAttributes`

File: creatingGame.js [hunk 3]
```diff
@@ -16,46 +16,59 @@ function matriz() {
...
+
   let padre = document.querySelector(".padre");
   padre.addEventListener("click", insertarX);
 
   var turno = jugador1;
+  let playCounter = 0;
 
   function insertarX(evento) {
+    playCounter = playCounter + 1;
     let imagen = evento.target;
```
Comment (erick2014) [inline creatingGame.js]: this should not be **let** but **constant** and should be name in english: `const image = event.target`

File: creatingGame.js [hunk 4]
```diff
@@ -16,46 +16,59 @@ function matriz() {
...
-      let columnElement = document.createElement("div");
-      let atributo2 = document.createAttribute("class");
-      atributo2.value = "column row" + [a + 1] + "-" + "column" + [b + 1];
-      columnElement.setAttributeNode(atributo2);
-      let imagen = document.createElement("img");
-      columnElement.appendChild(imagen);
-      rowElement.appendChild(columnElement);
-    }
-  }
+  let numRows = document.getElementById("filas").value;
```
Comment (erick2014) [inline creatingGame.js]: rename this variable to `rowsNumber`. Please translate all variables from spanish to english.

File: creatingGame.js [hunk 5]
```diff
@@ -16,46 +16,59 @@ function matriz() {
...
+  let numColumns = document.getElementById("columnas").value;
```
Comment (erick2014) [inline creatingGame.js]: rename this variable to `columnsNumber`

File: creatingGame.js [hunk 6]
```diff
@@ -16,46 +16,59 @@ function matriz() {
...
+  let game = drawGame(numRows, numColumns);
+  let scores = game.scoresMatriz;
+  let positionsCunter = game.positionsCunter;
+
+  let winnerMessage = "";
+
   let padre = document.querySelector(".padre");
   padre.addEventListener("click", insertarX);
 
   var turno = jugador1;
```
Comment (erick2014) [inline creatingGame.js]: turno should be in english: `player=player1`

**Review by erick2014 — CHANGES_REQUESTED** [review]

File: creatingGame.js [hunk 7]
```diff
@@ -1,74 +1,73 @@
...
-  let textojugador2 = document.createTextNode("JUGADOR O");
-  jugador2.appendChild(textojugador2);
-  players.appendChild(jugador2);
+  let player2 = document.createElement("div");
+  let attplayer2 = document.createAttribute("class");
+  attplayer2.value = "player2";
+  player2.setAttributeNode(attplayer2);
+  let textPlayer2 = document.createTextNode("JUGADOR O");
+  player2.appendChild(textPlayer2);
+  players.appendChild(player2);
```
Comment (erick2014) [inline creatingGame.js]: create a function ( in utils.js file) that creates these two texts( text for player 1, text for player 2 ). Just call the functions twice and pass each text that is going to be for each player.

**Review by erick2014 — CHANGES_REQUESTED** [review]

**Pull request merged**